keyboards/bear_face/info.json

```diff
@@ -0,0 +1,824 @@
+[
```

Comment:
This file has the wrong format <https://docs.qmk.fm/#/reference_configurator_support>. It looks like KLE so <https://qmk.fm/converter/> might help.